executor/tests/runtime_work/test_local_task_store.py

```
@@ -0,0 +1,1167 @@
+    from executor.runtime_work.rpc_handler import RuntimeWorkRpcHandler
+
+    codex_home = tmp_path / "codex-home"
+    thread_id = "019ee7f6-456a-78a1-96b1-66451afc310e"
+    discovery, _fake_codex = _codex_discovery_for_threads(
+        codex_home,
+        SimpleNamespace(
+            id=thread_id,
+            cwd="/repo/Wegent",
+            name=None,
+            preview="hi",
+            path="/tmp/thread.jsonl",
```

Ruff (0.15.17)

[error] Probable insecure usage of temporary file or directory: "/tmp/thread.jsonl"

(S108)